### PR TITLE
couchdb update

### PR DESCRIPTION
**Pull request by bindushree16**
https://github.com/dry-dock/c7all/issues/30

image

### CODE REVIEW

File: version/couchdb_2_1_1.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
 #!/bin/bash -e
-echo "================= Installing Cassandra 2.1.1 ==================="
+echo "================= Installing Couchdb 2.1.1 ==================="
 echo "[bintray--apache-couchdb-rpm]
 name=bintray--apache-couchdb-rpm
 baseurl=http://apache.bintray.com/couchdb-rpm/el7/x86_64/
 gpgcheck=0
 repo_gpgcheck=0
 enabled=1" > /etc/yum.repos.d/bintray-apache-couchdb-rpm.repo
+#couchbd is failing if its on this path so moving it to other location
```
Comment (ric03uec) [inline version/couchdb_2_1_1.sh]: can we add the link to the page thats suggests this alternative

Comment (bindushree16) [reply version/couchdb_2_1_1.sh]: ok sure

File: version/couchdb_2_1_1.sh [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+echo "================= Installing Couchdb 2.1.1 ==================="
 echo "[bintray--apache-couchdb-rpm]
 name=bintray--apache-couchdb-rpm
 baseurl=http://apache.bintray.com/couchdb-rpm/el7/x86_64/
 gpgcheck=0
 repo_gpgcheck=0
 enabled=1" > /etc/yum.repos.d/bintray-apache-couchdb-rpm.repo
+# couchbd is failing if its on this path so moving it to other location 
+# https://www.spinics.net/lists/centos/msg169867.html
+mv /usr/lib/python2.7/site-packages/urllib3/packages/ssl_match_hostname  /usr/lib/python2.7/site-packages/urllib3/packages/ssl_match_hostname.backup
```
Comment (ric03uec) [inline version/couchdb_2_1_1.sh]: is urllib3 working after this change? we need to make sure that whatever package was installing it is working correctly

Comment (bindushree16) [reply version/couchdb_2_1_1.sh]: ya its working fine now  its printing correct version

image

Comment (ric03uec) [reply version/couchdb_2_1_1.sh]: im asking is urllib3 working, not couchdb. we're changing the files in urllib3 and there are packages like apachelibcloud that depend on it. how are we verifiying that those are working correctly?

Comment (bindushree16) [reply version/couchdb_2_1_1.sh]: i used the script which uses urllib3 in apachelibcloud .this is working fine

image

Comment (ric03uec) [reply version/couchdb_2_1_1.sh]: ok. looks good

**Pull request merged**